pkg/virt-api/webhooks/mutating-webhook/mutators/namespace-limits.go

```diff
@@ -44,10 +44,10 @@ func isResourceRequirementMissing(resourceRequirements kubev1.ResourceRequiremen
 	return memoryOrCpuIsMissing(resourceRequirements.Limits) || memoryOrCpuIsMissing(resourceRequirements.Requests)
 }
 
-func applyNamespaceLimitRangeValues(vmi *kubev1.VirtualMachineInstance, limitrangeInformer cache.SharedIndexInformer) {
+func applyNamespaceLimitRangeValues(vmi *kubev1.VirtualMachineInstance, limitrangeInformer cache.SharedIndexInformer, namespace string) {
 	// Copy namespace limits (if exist) to the VM spec
 	if isResourceRequirementMissing(vmi.Spec.Domain.Resources) {
```

Comment:
Are those limits always specified on the namespace? I assume yes, but worth asking :)

Reply:
No, but that's alright as we will just not apply anything.